packages/docs/docs/plugin/official/plugin-pwa.md

```diff
@@ -22,6 +22,37 @@ module.exports = {
 }
 ```
 
+::: tip PWA NOTES
+To make your site fully PWA-compliant, you will need to:
+
+- provide a Web App Manifest and icons in `.vuepress/public`
```

Comment:
```suggestion
- provide a web app manifest and icons in `.vuepress/public`,
```

Reply:
@sarahdayan are you sure it's correct to add punctuation for items list?

Reply:
It depends, there are several schools of thought. Here's a good reference of [different use cases](http://www.plainenglish.co.uk/punctuating-bulleted-lists.html).

In this case, this is a sentence broken down into several items.